app/scripts/controllers/user-storage/encryption.ts

```diff
@@ -0,0 +1,117 @@
+import sjcl from 'sjcl';
+
+// Encryption / Decryption taken from - https://github.com/luke-park/SecureCompatibleEncryptionExamples/blob/master/TypeScript/SCEE-sjcl.ts
+class EncryptorDecryptor {
+  #BITS_PER_WORD = 32;
```

Comment:
Is there a reason for using words instead of bits and bytes? IMHO it makes the code a bit more diffilcult to grasp.